src/common/utils.js

```diff
@@ -537,3 +537,19 @@ export const compareVersions = makeNiceCompare((v1, v2) => {
 
 export const isXosanPack = ({ name }) =>
   startsWith(name, 'XOSAN')
+
+// ===================================================================
+
+export const getCoresPerSocketPossibilities = (maxCoresPerSocket, vCPUs) => {
+  // According to : https://www.citrix.com/blogs/2014/03/11/citrix-xenserver-setting-more-than-one-vcpu-per-vm-to-improve-application-performance-and-server-consolidation-e-g-for-cad3-d-graphical-applications/
+  const maxVCPUs = 16
+
+  const options = []
+  if (maxCoresPerSocket !== undefined && vCPUs !== '') {
+    for (let coresPerSocket = maxCoresPerSocket; coresPerSocket >= vCPUs / maxVCPUs; coresPerSocket--) {
```

Comment:
As a general rule: avoid doing computations in the loop condition.

Use a variable if you have to.